ansible/README.md

```diff
@@ -0,0 +1,121 @@
+Deploying Openwhisk using Ansible
+=========
+
+
+### Getting started
+
+If you want to deploy Openwhisk locally using Ansible, you first need to install Ansible on your development environment:
+
+#### Ubuntu users
+```
+sudo apt-get install python-pip
+sudo pip install ansible==2.0.2.0
+```
+
+#### Vagrant users
+Nothing to be done, Ansible is already installed during vagrant provisioning.
+
+#### Mac users
+It is assumed that a VM has been provisioned using **docker-machine**.
+
+```
+brew install python
+pip install ansible==2.0.2.0
+
+cd ansible
+ansible-playbook setup.yml
```

Comment:
we did not document at first because the original tweak-dockermachine.sh had the machine name hard-coded. with your proposed change there we can now add the option as the user can now change the name of the machine.